UnitySDK/Assets/ML-Agents/Scripts/Academy.cs

```diff
@@ -21,10 +22,16 @@
 
 namespace MLAgents
 {
+    public class AcademyStepper : MonoBehaviour
+    {
+        void FixedUpdate()
```

Comment:
TODO hide?

Reply:
Why hide ?
Also, maybe rename AcademyDefaultStepper ? Or AcademyFixedUpdateStepper for clarity ?

Reply:
Renamed to AcademyFixedUpdateStepper.

I added `m_StepperObject.hideFlags = HideFlags.HideInHierarchy` in the setup code below, because I think it would be surprising to the user to see a GameObject that they didn't create there.

Reply:
Maybe only hide if the user is not in using a DEBUG flag ? Like what you did for sensor validators?

Reply:
There's really no reason for the user to interact with this object in the editor, or ever.  I'm in favor of hiding it over showing it conditionally (which would only ever be in the editor in debug mode).